### PR TITLE
Changing from old STM CLA to the new DCO

### PR DESCRIPTION
**Pull request by jbech-linaro**
`Signed-off-by: Joakim Bech <joakim.bech@linaro.org>`

### CODE REVIEW

File: Notice.md [hunk 1]
```diff
@@ -1,35 +1,4 @@
...
-
-Contributions must be original work of the contributor. In order to preserve the rights of the contributor while allowing distribution to and protection of the recipients of OP-TEE, the contributor must complete, sign and send the Contribution Agreement or a scanned copy to ST for counter-signature, prior to any contribution. The address where to send the agreement and other details will be provided upon contact with the OP-TEE gatekeepers.
-Once the Contribution Agreement is complete, the contributor may propose contributions to the OP-TEE gatekeepers. Proposed Contributions are reviewed for acceptance by the OP-TEE gatekeepers and the OP-TEE community.
-
-Submission of non-original work
-------
-
-You may submit work that is not your original creation separately from any Contribution, identifying the complete details of its source and of any license or other restriction of which you are personally aware. Such submissions are not subject to the Contribution Agreement. They are reviewed for acceptance by the OP-TEE gatekeepers and the OP-TEE community.
+To avoid duplicating information we just link to the Notice.md file in optee_os
```
Comment (jenswi-linaro) [inline Notice.md]: "just" isn't needed

File: Notice.md [hunk 2]
```diff
@@ -1,35 +1,4 @@
...
+which states the contributor agreement rules etc, I.e, optee_client follows what
```
Comment (jenswi-linaro) [inline Notice.md]: `s/I.e/i.e/`

File: Notice.md (hunk 1 above)
Comment (jbech-linaro) [inline Notice.md]: Will fix

File: Notice.md (hunk 2 above)
Comment (jbech-linaro) [inline Notice.md]: Will fix

**Comment by pascal-brand-st-dev** [comment]
Reviewed-by: Pascal Brand <pascal.brand@linaro.org>

**Comment by jenswi-linaro** [comment]
```
Reviewed-by: Jens Wiklander <jens.wiklander@linaro.org>
```

**Comment by jbech-linaro** [comment]
Rebased and tags applied.

**Pull request merged**